src/main/java/com/synopsys/integration/detect/workflow/componentlocationanalysis/GenerateComponentLocationAnalysisOperation.java

```diff
@@ -1,25 +1,29 @@
 package com.synopsys.integration.detect.workflow.componentlocationanalysis;
 
+import com.google.common.collect.ImmutableSet;
 import com.google.gson.JsonObject;
 import com.synopsys.integration.blackduck.api.generated.view.DeveloperScansScanView;
 import com.synopsys.integration.detect.configuration.DetectUserFriendlyException;
 import com.synopsys.integration.detect.workflow.bdio.BdioResult;
 import com.synopsys.integration.componentlocator.ComponentLocator;
 import com.synopsys.integration.componentlocator.beans.Component;
 import com.synopsys.integration.componentlocator.beans.Input;
+import com.synopsys.integration.detector.base.DetectorType;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import com.synopsys.integration.detect.workflow.report.util.ReportConstants;
 
 import java.io.File;
 import java.util.List;
+import java.util.Set;
 
 /**
  * This class will generate the appropriate input file for Component Locator, invoke the library's obfuscated JAR and
  * save the resulting output file in the appropriate output subdirectory.
  */
 public class GenerateComponentLocationAnalysisOperation {
     public static final String DETECT_OUTPUT_FILE_NAME = "components-with-locations.json";
+    public static final Set<DetectorType> SUPPORTED_DETECTORS = ImmutableSet.of(DetectorType.NPM, DetectorType.MAVEN, DetectorType.GRADLE, DetectorType.NUGET);
```

Comment:
This set should come from the library? If we release two versions of library, one support more types - then Detect would always get the correct data.

Reply:
Done. Please see related CLL merge request that will need to be merged, version released, update version in build.gradle of Detect. all before this PR can be merged: https://sig-gitlab.internal.synopsys.com/integrations-detect/fixpr/component-locator/-/merge_requests/19

Reply:
@niravrsynopsys ^

Reply:
I have approved MR 19 of CLL. @niravrsynopsys or @dterrysynopsys  please review.

Reply:
Please note the build.gradle in this PR assumes MR # 19 changes will go in CLL v1.0.7